### DISCUSSION TITLE
Is it allowed to have multiple accounts for different projects?

### DISCUSSION THREAD

**Discussion by jemikanegara** [planetscale/discussion#357] [Q&A] [answered]
I want to create multiple db for different projects, is it okay to create another accounts for this purpose? if not, I will probably just prefix my table name

**Comment by fooforge** [comment, marked as answer]
Hey @jemikanegara,

Great question!

A user on PlanetScale can create up to three organizations with one free database in each. If you need more than three, we can up the organizations limit if you have at least one database on our [Scaler plan](https://planetscale.com/pricing). Reach out to us via [our support portal](https://support.planetscale.com/) and we can talk about it further.

You can, obviously, also create multiple accounts under different email addresses, we're not going to hold it against you. However, our free offering is mainly meant for Hobby purposes; think Open Source development, small side projects, or students learning programming and SQL. And, PlanetScale only thrives with users paying for our services.
As such, I would want to encourage you to use our free offering intelligently, e.g. only use as many free database as you need, [put them to sleep](https://planetscale.com/docs/concepts/database-sleeping#what-is-database-sleeping) or delete them if they're no longer needed, and upgrade to Scaler if one of your projects turns into a small business.
